src/utils/operators/debounce.ts

```diff
@@ -12,4 +12,4 @@ export const debounce = (func: Function, wait: number, immediate?: any) => {
     timeout = setTimeout(later, wait);
     if (callNow) func.apply(context, args);
   };
-}
```

Comment:
I don't understand what it does to be honest and why vars? Not let or consts?

Reply:
Ah sorry. Its copy paste method from lodash lib. It allows call only one function and Block execution of next ones.